src/widgets/_checkbuttons.scss

```diff
@@ -62,6 +62,15 @@ checkbutton {
             margin-right: rem(6px);
         }
     }
+
+    &.image-button:checked {
```

Comment:
```suggestion
    &.image-button {
        // Invisible stroke to prevent things from moving when :checked
        border: 1px solid transparent;
    }

    &.image-button:checked {
```